tests/test_decorators.py

```diff
@@ -135,11 +136,50 @@ def test_cognito_login_cookie_samesite(client, cfg, access_token, token_response
         assert "SameSite=Strict" in response.headers["Set-Cookie"]
 
 
+def test_cognito_refresh_missing_token(
+    client, cfg, access_token, refresh_token_response
```

Comment:
```suggestion
    client, cfg, access_token, refresh_token_response,
```

same here, adding a trailing comma on the final arg